Cargo.toml

```diff
@@ -137,6 +137,7 @@ url = "2.2"
 uuid = { version = "1.2", features = ["v4", "serde"] }
 walkdir = "2.3.2"
 webpki-roots = "0.23"
+wiremock = "0.5.19"
```

Comment:
Is there a benefit to adidng this as a workspace dependency? (I was initially going to nag that this should be a proxy OR dev-dep but it is referenced under proxy/Cargo.toml under dev-dep.)

Reply:
I just followed the default "everything is a workspace dependency" that we seem to have. Happy to challenge that though, or even get rid of wiremock entirely